config/config.go

```diff
@@ -0,0 +1,132 @@
+package config
```

Comment:
I extracted the config to use it inside `test/mock `and inside `mongodbatlas`. This avoid the cycle dependencies errors